packages/vuetify/src/composables/border-radius.ts

```diff
@@ -0,0 +1,61 @@
+// Utilities
+import { computed } from 'vue'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+import type { PropType } from 'vue'
+
+export type BorderRadiusProp = boolean | 0 | '0' | '' | 'xs' | 'sm' | 'lg' | 'xl' | 'circle' | 'pill' | 'shaped' | 'tile' | null
+
+export interface BorderRadiusProps {
+  rounded?: BorderRadiusProp
+}
+
+const radius: BorderRadiusProp[] = [
+  true,
+  false,
+  0,
+  '0',
+  'xs',
+  'sm',
+  'lg',
+  'xl',
+  'circle',
+  'pill',
+  'shaped',
+  'tile',
+]
```

Comment:
Instead of having to hard-code types by hand like `type BorderRadiusProp = boolean | 0 // ...` above, [what I was trying to do here was to use this array here to create a type](https://github.com/vuetifyjs/vuetify/pull/12801/commits/3acaffb0a60e009810ec22e0a5f41d711238ebc0#r546802674).

```ts
const radius = [
  true,
  false,
  0,
  '0',
  'xs',
  'sm',
  'lg',
  'xl',
  'circle',
  'pill',
  'shaped',
  'tile',
] as const

export type BorderRadiusProp = typeof radius[number]
```